config/routes.rb

```diff
@@ -333,6 +333,7 @@
       get '/hybrid_mobile/document_capture' => 'hybrid_mobile/document_capture#show'
       put '/hybrid_mobile/document_capture' => 'hybrid_mobile/document_capture#update'
       get '/hybrid_mobile/capture_complete' => 'hybrid_mobile/capture_complete#show'
+      get '/hybrid_handoff' => 'hybrid_handoff#show'
```

Comment:
Do you need a corresponding PUT route to handle the phone form submission?

Reply:
This ticket is only for the show/get action. The form submission will come in a subsequent ticket